server/src/main/java/com/google/musicanalysis/site/YoutubeGenres.java

```diff
@@ -0,0 +1,104 @@
+package com.google.musicanalysis.site;
+
+import com.google.gson.JsonParser;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import java.lang.Math;
+
+enum MusicTopic {
+  MUSIC_ONLY,
+  MUSIC_SUBGENRE,
+  NOT_MUSIC
+}
+
+/** contains final object that YoutubeServlet.java sends to frontend */
+public class YoutubeGenres {
+  private HashMap<String, Integer> genreData = new HashMap<String, Integer>();
+  protected int totalLiked;
```

Comment:
nit: this should be above the private variables with a newline inbetween and a comment (since it's now part of your public api).